lib/rules/selector-anb-no-unmatchable/__tests__/index.mjs

```diff
@@ -43,112 +45,163 @@ testRule({
 			code: 'a:nth-child(3n + - 6) {}',
 			description: 'CSSTree parse error not flagged',
 		},
+		{
+			code: stripIndent`
+				/* stylelint-disable-next-line selector-anb-no-unmatchable */
+				a:nth-child(0), /* stylelint-disable-next-line selector-anb-no-unmatchable */
+				b:nth-child(0),
+				/* stylelint-disable-next-line selector-anb-no-unmatchable */
+				c:nth-child(0)
+				{}
+			`,
+		},
 	],
 
 	reject: [
 		{
 			code: 'a:nth-child(0) {}',
 			message: messages.rejected(':nth-child'),
 			line: 1,
-			column: 3,
+			column: 2,
 			endLine: 1,
-			endColumn: 16,
+			endColumn: 15,
```

Comment:
[suggestion] Should we align the reported range and the message? For example, in this case, should the column range be `2`-`12` (`:nth-child`) instead of `3`-`15` (`nth-child(0)`)?

```css
a:nth-child(0) {}
 ↑         ↑
123456789012345678

/* Unexpected unmatchable An+B selector ":nth-child" */
```

Reply:
Or should we change the message so that it includes the problematic selector and not just the pseudo name?

`:nth-child` -> `:nth-child(0)`

Given that the pseudo is incorrect because specific arguments are used it might be better to include those arguments in the message?

Reply:
Make sense. Let's change the message to include arguments. 👍🏼